src/components/TextDiv/TextDiv.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import tag from 'clean-tag';
+import userSpacing from 'extensions/userSpacing';
+
+/**
+ * Text divs are vertically aligned based on the cap and
+ * base lines of the text. They support the `spacing`
+ * property, which allows a user to specify custom spacing
+ * around the component.
+ *
+ * **NOTE:** Many components provide the `spacing` property
+ * already. This component is provided as a catch-all
+ * solution.
+ */
+const TextDiv = styled(tag.div).attrs({
```

Comment:
I don't think we should really encourage use of this externally at all. Semantically, if you mean to put a paragraph, you should be using `P`. Not that I don't think we're all aware of that practice, but I have noticed that bad practices tend to get adopted quickly with such a wide-spread resource as this library.

Basically I think a nice big `USE P INSTEAD IF YOU NEED A PARAGRAPH OF TEXT` would be a good addition, or even maybe not documenting this in the site.

Reply:
The reason I added this was because of the problems with anchors (specifically, `inline-block` not accepting negative margins). I can wrap it in a `<P>`, but it's not really a block of text.

Reply:
Yeah, it makes sense for internal use, I just worry about it becoming a temptation to lazy element selection if advertised as a library feature.

Reply:
I guess you mean you want to use it externally, though. I really feel like there should be a way to more seamlessly fix the anchors. Like...

```
display: block;
margin: ${props => props.spacing};

p > &,
span > & {
  display: inline;
  margin: auto;
}
```

Reply:
I can try that, though it might be confusing to. change the behavior based on the parent .